integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/VersionSpecificVisitorsProvider.java

```diff
@@ -0,0 +1,25 @@
+package io.openlineage.spark.agent.lifecycle;
+
+import io.openlineage.spark.agent.lifecycle.plan.VisitorFactory;
+import org.apache.spark.sql.SparkSession;
+
+public class VersionSpecificVisitorsProvider {
```

Comment:
If the version-specific factory extends the common one, we can make this simply `VisitorFactoryProvider`

Reply:
sure